projects/packages/codesniffer/tests/php/tests/files/default.xml

```diff
@@ -2,7 +2,7 @@
 <ruleset>
 
 	<config name="minimum_supported_wp_version" value="6.0" />
```

Comment:
Again, I'd bump all these to at least 6.3 too.

Reply:
Done in 51746059f9565019d372d6bcfe6dce647076ea8e